code/modules/antagonists/changeling/powers/gaseous_pores.dm

```diff
@@ -0,0 +1,23 @@
+/datum/action/changeling/gaseous_pores
+	name = "Gaseous Pores"
+	desc = "Our skins bursts, releasing noxious gases to confuse opponents and cover our retreat."
+	helptext = "We are still affected by the gases we emit, internals are recommended.Functions while unconscious."
+	button_icon_state = "smoke"
+	chemical_cost = 35
+	dna_cost = 2
+	req_stat = UNCONSCIOUS
+	var/silent = FALSE
+
+/datum/action/changeling/gaseous_pores/sting_action(mob/user)
+	..()
+	var/turf = get_turf(user)
```

Comment:
```suggestion
	var/turf/T = get_turf(user)
	if(!T)
		return FALSE
```

Reply:
What purpose does this serve? just curious